.bin/check_and_load_xml.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+# Encode non UTF-8 files to UFT-8 for Postgres compatibility
+
+database=${PGDATABASE:-ggircs_dev}
+input_file="$1"
+encoded_file=${input_file/xmls_from_eccc/utf8_encoded_xmls}
+filename=$(basename -- "$1")
+
+_psql() {
+  user=${PGUSER:-$(whoami)}
+  database=${PGDATABASE:-$database}
+  host=${PGHOST:-localhost}
+  port=${PGPORT:-5432}
```

Comment:
Given these are the [libpq-env-vars](https://www.postgresql.org/docs/current/libpq-envars.html) shouldn't we be able to simply:

 ```bash
  PGUSER="${PGUSER:-$(whoami)}"
  PGDATABASE="${PGDATABASE:-$database}"
  PGHOST="${PGHOST:-localhost}"
  PGPORT="${PGPORT:-5432}"
```

Reply:
...and if so, wouldn't `${PGDATABASE:-ggircs_dev}` be the only thing in there that diverges from defaults?